### PR TITLE
fuzzy finder: add docs

### PR DESCRIPTION
**Pull request by taras-yemets**
Adds fuzzy finder docs.

image

Adding fuzzy finder to the code search explanation section instead of code-nav explanation section as in my opinion it better fits the rest of the content (cc: @felixfbecker). 

## Test plan
Tested manually.

### CODE REVIEW

File: doc/code_search/explanations/features.md [hunk 1]
```diff
@@ -98,6 +98,13 @@ Your site admin can enable search contexts on your private instance in **global
 }
 ```
 
+## Fuzzy search
+
+Fuzzy finder helps you find any item by its name in the current repository or everywhere. You can search for repositories, symbols, files and files. 
+To open the fuzzy finder, press `Cmd/Ctrl+K` anywhere on the Sourcegraph web app.
+
+<img src="https://i.imgur.com/ErZOOMG.png" alt="Fuzzy search">
```
Comment (felixfbecker) [inline doc/code_search/explanations/features.md:106]: Could we use our `sourcegraph-assets` GCS bucket instead of imgur?

**Review by felixfbecker — COMMENTED** [review]
👏🏻 for adding docs.

File: doc/code_search/explanations/features.md [hunk 2]
```diff
@@ -98,6 +98,13 @@ Your site admin can enable search contexts on your private instance in **global
 }
 ```
 
+## Fuzzy search
+
+Fuzzy finder helps you find any item by its name in the current repository or everywhere. You can search for repositories, symbols, files and files. 
+To open the fuzzy finder, press `Cmd/Ctrl+K` anywhere on the Sourcegraph web app.
```
Comment (olafurpg) [inline doc/code_search/explanations/features.md:104]:
```suggestion
To open the fuzzy finder, press `Cmd+K` (macOS) or `Ctrl+K` (Linux/Windows) from any page.
```

File: doc/code_search/explanations/features.md [hunk 3]
```diff
@@ -98,6 +98,13 @@ Your site admin can enable search contexts on your private instance in **global
 }
 ```
 
+## Fuzzy search
+
+Fuzzy finder helps you find any item by its name in the current repository or everywhere. You can search for repositories, symbols, files and files.
```
Comment (olafurpg) [inline doc/code_search/explanations/features.md:103]:
```suggestion
Use the fuzzy finder to quickly navigate to a repository, symbol, or file.
```

File: doc/code_search/explanations/features.md [hunk 4]
```diff
@@ -98,6 +98,13 @@ Your site admin can enable search contexts on your private instance in **global
 }
 ```
 
+## Fuzzy search
+
+Fuzzy finder helps you find any item by its name in the current repository or everywhere. You can search for repositories, symbols, files and files. 
+To open the fuzzy finder, press `Cmd/Ctrl+K` anywhere on the Sourcegraph web app.
+
+<img src="https://i.imgur.com/ErZOOMG.png" alt="Fuzzy search">
+
```
Comment (olafurpg) [inline doc/code_search/explanations/features.md:107]: Use the "Searching everywhere" or "Searching in this repo" filter to determine whether to search for results only in the active repository or globally.

SCREENSHOT.

File: doc/code_search/explanations/features.md (hunk 2 above)
Comment (olafurpg) [reply doc/code_search/explanations/features.md:104]: Use the dedicated Repos, Symbols, and Files tabs to search only for a repository, symbol, or file. Each tab has a dedicated shortcut:

* Repos: Cmd+I (macOS), Ctrl+K (Linux/Windows)
* Symbols: Cmd+O (macOS), Cmd+Shift+O (macOS Safari), Ctrl+O (Linux/Windows)
* Files: Cmd+P (macOS), Ctrl+P (Linux/Windows)

**Review by olafurpg — CHANGES_REQUESTED** [review]
+1 to @felixfbecker's comment along with few other changes

**Comment by taras-yemets** [comment]
@felixfbecker, @olafurpg, 
Thank you for your feedback!

Addressed it and added experimental badge with a note linking to the [feedback issue](https://github.com/sourcegraph/sourcegraph/discussions/42874).

Screenshot 2022-12-12 at 14 48 38

**Review by philipp-spiess — APPROVED** [review]

**Review by olafurpg — APPROVED** [review]

**Pull request merged**